package.json

```diff
@@ -148,12 +148,12 @@
       },
       {
         "view": "watermelonExplorerTreeProvider",
-        "contents": "Watermelon 🍉 will help you understand code context better. \n [Get Code Context](command:watermelon.start)\n",
+        "contents": "[Get Code Context](command:watermelon.start) \n 🍉 About Watermelon \n Watermelon helps your team merge Pull Requests With Business Value. \n You can also contextualize your pull requests with our [GitHub Application](https://github.com/apps/watermelon-context) \n [Add Watermelon to the repo's recommended extensions](command:watermelon.recommend). \n Support our Open Source work by [Starring us on Github](https://github.com/watermelontools/wm-extension). \n Need help? [Join us on Discord](https://discord.gg/H4AE6b9442) \n [Read our Blog](https://www.watermelontools.com/blog) and learn with us.",
         "when": "watermelon.isLogged && !watermelon.isLoading"
```

Comment:
Long text for a loading screen of up to a second.

Reply:
I'll leave it with the previous length